core/to_from_v8.rs

```diff
@@ -1,5 +1,8 @@
 // Copyright 2018-2024 the Deno authors. All rights reserved. MIT license.
 
+use crate::{error::StdAnyError, runtime::ops};
+use std::convert::Infallible;
+
 pub trait ToV8<'a> {
```

Comment:
While we're here, let's add some basic docs to ToV8 and FromV8.

Reply:
Added some docs, feel free to review